packages/itwinui-react/src/core/Tree/TreeNode.tsx

```diff
@@ -15,23 +15,43 @@ type TreeNodeProps = {
    * It has to be compatible with HTML id attribute.
    */
   nodeId: string;
+  /**
+   * Props for treeNode
+   */
+  nodeProps?: React.ComponentProps<any>;
```

Comment:
These should all be `'div'`, `'span'`, etc (matching the corresponding elements) rather than `any`

Reply:
Fixed in this [commit](https://github.com/iTwin/iTwinUI/pull/2022/commits/54fc0932a2836605be25ecaba9749ecc4720c1b9).

Reply:
Added in this [commit](https://github.com/iTwin/iTwinUI/pull/2022/commits/5ef04937148b9d722e75d364ad109f8c539af188).